packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -438,7 +439,7 @@ export async function createEmulator(
     ["hw.cpu.ncore", "4"],
     ["hw.dPad", "no"],
     ["hw.device.manufacturer", "Google"],
-    ["hw.device.name", deviceName],
+    ["hw.device.name", mapModelToId(modelName)],
```

Comment:
I wonder if this is even necessary to have a proper device name there, have you tested this? Maybe there's no reason to introduce this mapping and this additional complexity here if it doesn't make any difference what we put there

Reply:
@km1chno tested this in https://github.com/software-mansion/radon-ide/pull/601 and without this prop device displayed wrong status bar layout